manifests/0000_30_07-certified-operators.configmap.yaml

```diff
@@ -1152,46 +1166,40 @@ data:
             - kind: Pod
               name: ''
               version: v1
-      
             specDescriptors:
             - description: Number of members in this Replica Set.
               displayName: Members
               path: members
               x-descriptors:
               - 'urn:alm:descriptor:text'
-      
             - description: Credentials for Ops Manager or Cloud Manager.
               displayName: Credentials
               path: credentials
               x-descriptors:
               - 'urn:alm:descriptor:io.kubernetes:core:v1:Secret'
-      
             - description: Project this deployment belongs to.
               displayName: Project
               path: project
               x-descriptors:
               - 'urn:alm:descriptor:io.kubernetes:core:v1:ConfigMap'
-      
             - description: MongoDB version to be installed.
               displayName: Version
               path: version
               x-descriptors:
               - 'urn:alm:descriptor:com.tectonic.ui:label'
-      
             - description: The service where this MongoDB deployment will be listening.
               displayName: Service
               path: service
               x-descriptors:
               - 'urn:alm:descriptor:io.kubernetes:Service'
-      
           - name: mongodbshardedclusters.mongodb.com
             description: MongoDB Sharded Cluster Deployment
             displayName: MongoDB Sharded Cluster
             group: mongodb.com
             kind: MongoDbShardedCluster
             version: v1
-      
             resources:
+            # FIXME(alecmerdler): These are probably supposed to be different resources for the different CRD configurations
```

Comment:
Do we want this in our release artifacts?

Reply:
I think so?